src/xia2/cli/cluster_analysis.py

```diff
@@ -40,32 +40,37 @@
     .help = 'Display the dendrogram with a log scale'
 }
 
-run_cluster_identification = True
-  .type = bool
-  .short_caption = "If True, in addition to running clustering analysis, identify"
-                   "clusters of interest for further analysis."
-
-max_cluster_height_difference = 0.5
-  .type = float
-  .short_caption = "Maximum hight difference between clusters"
-max_output_clusters = 10
-  .type = int
-  .short_caption = "Maximum number of important clusters to be output"
-min_cluster_size = 5
-  .type = int
-  .short_caption = "Minimum number of datasets for an important cluster"
-output_correlation_cluster_number = 0
-  .type = int
-  .short_caption = "Option to output a specific correlation cluster when re-running the code"
-output_cos_cluster_number = 0
-  .type = int
-  .short_caption = "Option to output a specific cos cluster when re-running the code"
-exclude_correlation_cluster_number = 0
-  .type = int
-  .short_caption = "Option to output all data excluding a specific correlation cluster"
-exclude_cos_cluster_number = 0
-  .type = int
-  .short_caption = "option to output all data excluding a specific cos cluster"
+clustering
```

Comment:
The changes to the options are good in general, I think multiplex and cluster_analysis should include a shared phil scope for these options, that is defined only in once place, so that the behaviour is the same for both. That way the clustering achieved by setting particular options in multiplex could be replicated by running xia2.cluster_analysis on the scaled.expt and scaled.refl from multiplex.

Reply:
Have re-worked this to put the clustering phil scope only in xia2.cluster_analysis - ScaleAndMerge then imports this. Also re-worked xia2.cluster_analysis to function more like multiplex.